requirements/default.txt

```diff
@@ -1,5 +1,5 @@
-numpy>=1.19
-scipy>=1.5,!=1.6.1
-pandas>=1.1
-networkx>=2.5
+networkx>=2.2,<3.0
+numpy>=1.19.0,<1.22.0
+scipy>=1.5.0,<1.7.0
+pandas>=0.23.4,<1.0.0
```

Comment:
Question: is it necessary to upper bound these versions? That's usually done if a newer version of one of these libraries will break our code in some way.

Reply:
I was having trouble getting the documentation to build when I didn't, but I think I might have resolved the issue.

Reply:
Also to your point, it would be nice to automate the up-versioning and pushing to pip

Reply:
I ended up needing to upper bound numpy, but that's the only necessary upper bound

Reply:
> I ended up needing to upper bound numpy, but that's the only necessary upper bound

Great! Could you please document what is the problem when building the docs as a new issue? We should try to figure that out.